python/lsst/skymap/baseSkyMap.py

```diff
@@ -166,6 +170,18 @@ def __iter__(self):
     def __len__(self):
         return len(self._tractInfoList)
 
+    def __hash__(self):
+        return hash(self._sha1)
```

Comment:
Here and below shouldn't it be `getSha1()` that is called. Otherwise how do you know it's been cached?